src/main/scala/encry/view/state/UtxoState.scala

```diff
@@ -102,13 +102,10 @@ class UtxoState(override val version: VersionTag,
   def proofsForTransactions(txs: Seq[EncryBaseTransaction]): Try[(SerializedAdProof, ADDigest)] = {
     log.debug(s"Generating proof for ${txs.length} transactions ...")
     val rootHash: ADDigest = persistentProver.digest
-    if (txs.isEmpty) {
-      Failure(new Exception("Got empty transaction sequence"))
-    } else if (!storage.version.exists(_.sameElements(rootHash))) {
+    if (txs.isEmpty) Failure(new Exception("Got empty transaction sequence"))
+    else if (!storage.version.exists(_.sameElements(rootHash)))
       Failure(new Exception(s"Invalid storage version: ${storage.version.map(Algos.encode)} != ${Algos.encode(rootHash)}"))
```

Comment:
Do you mean here to make a new instance of class Exception or to throw a new Exception? If second this code is wrong. def proofsForTransactions can return Failure to val (adProof: SerializedAdProof, adDigest: ADDigest) = state.proofsForTransactions(txs).get.

Reply:
It was implied to work with exceptions within `Try` here if the failure occurred. `.get` from miner is already removed in another pr.